src/main/java/technology/dice/dicefairlink/discovery/tags/awsapi/ResourceGroupApiTagDiscovery.java

```diff
@@ -0,0 +1,72 @@
+package technology.dice.dicefairlink.discovery.tags.awsapi;
+
+import software.amazon.awssdk.services.resourcegroupstaggingapi.ResourceGroupsTaggingApiClient;
+import software.amazon.awssdk.services.resourcegroupstaggingapi.model.GetResourcesRequest;
+import software.amazon.awssdk.services.resourcegroupstaggingapi.paginators.GetResourcesIterable;
+import technology.dice.dicefairlink.config.FairlinkConfiguration;
+import technology.dice.dicefairlink.discovery.tags.ExclusionTag;
+import technology.dice.dicefairlink.discovery.tags.TagFilter;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+public class ResourceGroupApiTagDiscovery implements TagFilter {
```

Comment:
Not tested? Looks like important bit of the logic.

Reply:
I don't have a way to test this at the moment.
Do you have any ideas ?